python-backend/app/api/permits/permit_amendment/models/permit_amendment.py

```diff
@@ -13,8 +13,8 @@
 
 class PermitAmendment(AuditMixin, Base):
     __tablename__ = 'permit_amendment'
-    permit_amendment_id = db.Column(db.Integer, primary_key=True, server_default=FetchedValue())
-    permit_amendment_guid = db.Column(UUID(as_uuid=True))
+    permit_amendment_id = db.Column(db.Integer, primary_key=True)
```

Comment:
I want to add a comment about how the id should be FetchedValue, but with the current test infrastructure that isn't possible.